tests/sample/ext/inc/test_ext_helpers.h

```diff
@@ -23,4 +23,9 @@ typedef int (*test_ebpf_extension_helper_function1_t)(test_program_context_t* co
 typedef void (*test_ebpf_extension_helper_function2_t)(void* memory_pointer, uint32_t size);
 #define test_ebpf_extension_helper_function2 ((test_ebpf_extension_helper_function2_t)TEST_EXT_HELPER_FN_BASE + 2)
 typedef void (*test_ebpf_extension_helper_function3_t)(uint8_t arg);
-#define test_ebpf_extension_helper_function3 ((test_ebpf_extension_helper_function3_t)TEST_EXT_HELPER_FN_BASE + 3)
\ No newline at end of file
+#define test_ebpf_extension_helper_function3 ((test_ebpf_extension_helper_function3_t)TEST_EXT_HELPER_FN_BASE + 3)
```

Comment:
I'd recommend using the style in ebpf_helpers.h with doxgen, the EBPF_HELPER() macro use, etc. Since this is supposed to be a sample, we want people to do extensions that have doxygen generated docs like https://microsoft.github.io/ebpf-for-windows/ebpf__helpers_8h.html has

Reply:
Done.

Reply:
not quite done.  This should use EBPF_HELPER() macro so the docs will be generated correctly. The EBPF_HELPER for the doxygen pass is defined in ./Doxyfile for you, you just have to use it.  Again, see the ebpf_helpers.h use of EBPF_HELPER and copy that.

Reply:
Sorry, I missed this suggestion. Are you asking me to make a copy of:

`
#ifndef __doxygen
#define EBPF_HELPER(return_type, name, args) typedef return_type(*name##_t) args
#endif
`

Reply:
yes and use it when defining the helpers, like ebpf_helpers.h does.

Reply:
Done.